Source/Krypton Components/Krypton.Toolkit/Controls Toolkit/KryptonComboBox.cs

```diff
@@ -1830,7 +1830,7 @@ public bool Sorted
     /// </summary>
     [Category(@"Data")]
     [Description(@"The items in the KryptonComboBox.")]
-    [Editor(@"System.Windows.Forms.Design.ListControlStringCollectionEditor", typeof(UITypeEditor))]
+    [Editor("System.Windows.Forms.Design.ListControlStringCollectionEditor, System.Design, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a", typeof(UITypeEditor))]
```

Comment:
Will probably need to check the other usages of the form designers as well.
And also in the EXT suite